crates/viewer/re_renderer_examples/framework.rs

```diff
@@ -332,7 +332,10 @@ pub fn load_rerun_mesh(re_ctx: &RenderContext) -> Vec<re_renderer::renderer::Mes
     let mut zipped_obj = zip.by_name("rerun.obj").unwrap();
     let mut obj_data = Vec::new();
     std::io::Read::read_to_end(&mut zipped_obj, &mut obj_data).unwrap();
-    re_renderer::importer::obj::load_obj_from_buffer(&obj_data, re_ctx).unwrap()
+    re_renderer::importer::obj::load_obj_from_buffer(&obj_data, re_ctx)
+        .unwrap()
```

Comment:
Can we return errors instead please to set a good example? using `anyhow` should be pretty easy, and probaly reduce line count (`?` is shorter than `.unwrap()`)